build.sh

```diff
@@ -132,30 +132,43 @@ cmake \
   $dir_root/bundle
 set +x
 
-# Build
-echo "Building ... `date`"
+# Install
+echo "Installing ... `date`"
 set -x
-if [[ $BUILD_JCSDA == 'YES' ]]; then
-  make -j ${BUILD_JOBS:-8} VERBOSE=$BUILD_VERBOSE
-else
-  builddirs="gdas iodaconv land-imsproc land-jediincr gdas-utils bufr-query da-utils"
-  for b in $builddirs; do
-    cd $b
-    set +x
-    echo "Building $b ... `date`"
-    set -x
-    make -j ${BUILD_JOBS:-8} VERBOSE=$BUILD_VERBOSE
-    cd ../
-  done
-fi
+make install -j ${BUILD_JOBS:-8}
 set +x
 
-# Install
-if [[ -n ${INSTALL_PREFIX:-} ]]; then
-  echo "Installing ... `date`"
-  set -x
-  make install -j ${BUILD_JOBS:-8}
-  set +x
+# If this is a workflow build, copy the installed files to the Global Workflow exec directory
+if [[ $WORKFLOW_BUILD == 'ON' ]]; then
+  echo "Copying installed files to Global Workflow directories ..."
+
+  # Make sure the required install directories exist in the Global Workflow
+  mkdir -p "$HOMEgfs"/exec
+  mkdir -p "$HOMEgfs/$CMAKE_INSTALL_LIBDIR"
+
+  # Move GDASApp executables from GDASApp to the Global Workflow
+  mv "$INSTALL_PREFIX/bin"/gdas* "$HOMEgfs/exec/"
+
+  # Move GDASApp executables (from submodules) that don't have a gdas_* prefix in the name
+  mv "$INSTALL_PREFIX/bin/bufr2ioda.x" "$HOMEgfs/exec/gdas_bufr2ioda.x"
+  mv "$INSTALL_PREFIX/bin/calcfIMS.exe" "$HOMEgfs/exec/gdas_calcfIMS.x" # .exe -> .x
+  mv "$INSTALL_PREFIX/bin/apply_incr.exe" "$HOMEgfs/exec/gdas_apply_incr.x" # .exe -> .x
+
+  # Move libraries from GDASApp to the Global Workflow
+  mv "$INSTALL_PREFIX/$CMAKE_INSTALL_LIBDIR"/* "$HOMEgfs/$CMAKE_INSTALL_LIBDIR/"
```

Comment:
See [comment](https://github.com/NOAA-EMC/global-workflow/pull/3760#issuecomment-2970542836) in g-w PR [#3760](https://github.com/NOAA-EMC/global-workflow/pull/3760). 

The use of `mv` proved problematic for me when rebuilding GDASApp on Hera.

A suggested change is provided in [this comment](https://github.com/NOAA-EMC/global-workflow/pull/3760#issuecomment-2970635701) from PR  [#3760](https://github.com/NOAA-EMC/global-workflow/pull/3760).

A bit more digging found perhaps an even better way to ensure user aliases don't interfere

> **Quote the command:**
Using single or double quotes around the command will also prevent alias expansion.
For example: "ls" or 'ls' will ignore any defined alias for ls